parameter_files/fates_params_default.cdl

```diff
@@ -951,6 +946,14 @@ data:
  fates_leaf_slatop = 0.012, 0.01, 0.024, 0.012, 0.03, 0.03, 0.012, 0.03, 
     0.03, 0.03, 0.03, 0.03 ;
 
+ fates_leaf_stomatal_intercept = 10000, 10000, 10000, 10000, 10000, 10000, 10000, 
```

Comment:
We planned to assign the value of "1000" for all PFTs. Do you have any thoughts on that?

Reply:
@Qianyuxuan,  I was planning on modifying the default values when we integrate your pull request. In this pull request, we are keeping the numbers the same so the tests produce the same results.

Reply:
@rgknox Okay! No more questions. Thanks.

Reply:
thanks for the review!

Reply:
Great, thanks @Qianyuxuan